src/score/figuredbasscontext.cpp

```diff
@@ -188,3 +120,64 @@ bool CAFiguredBassContext::remove(CAMusElement* elt)
 
     return success;
 }
+
+CAMusElement *CAFiguredBassContext::insertEmptyElement(int timeStart)
+{
+    CAFiguredBassMark *newElt = new CAFiguredBassMark(this, timeStart, 1);
+    int i;
+    for (i = 0; i < _figuredBassMarkList.size() && _figuredBassMarkList[i]->timeStart() < timeStart; i++)
+        ;
```

Comment:
I checked and this is old code, but was / is there a plan to add more code?
Else for loop could be removed.

Reply:
Actually, the for loop here is to compute the value of `i` (it's declared outside of the for block), because we have no more intelligent way of computing the position than linearly scanning the list. We could do binary search in the future though.